src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -265,14 +266,20 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
     ]
   );
 
+  const backgroundImageClass = `VenueRoomsEditor__background-image ${backgroundImageClassName}`;
+  const roomImageClass = (room: RoomData_v2) =>
```

Comment:
```suggestion
```

The above function will get re-created on each render. To avoid `useCallback`, there are 2 options:

1. Inline the call to `classNames` within the component/prop itself, example:
```tsx
            <img
              className={classNames("VenueRoomsEditor__room-image", {
                "VenueRoomsEditor__room-image--disabled": !room.isEnabled,
                "VenueRoomsEditor__room-image--enabled": room.isEnabled,
              })}
```

2. move it outide the component, just like `classNames` is:
```tsx
const backgroundImageClasses: (options: { isEnabled?: boolean }) => string = ({
  isEnabled,
}) =>
  classNames("VenueRoomsEditor__room-image", {
    "VenueRoomsEditor__room-image--disabled": !isEnabled,
    "VenueRoomsEditor__room-image--enabled": isEnabled,
  });

// and inside the component
            <img
              className={backgroundImageClasses(room)}

```

Reply:
fixed in a4aa1c4f4ec90d75594111a0ed9e49045dfe7cd6